funding-requests.md

```diff
@@ -0,0 +1,48 @@
+# Funding Requests
+
+Need to pay for something on behalf of the Kubernetes Project? This is the
+document to read. In general there are a few rough types of costs that can be
+covered:
+
+- Infrastructure: testing infra, video conferencing, mailing lists, domains, etc
+- Events: SIG face-to-face meetings, developer summits, etc
+- Consulting services: docs writers, security auditors, etc
+- Community Gifts: swag codes, thank you cards, etc
+
+## How to request funding
+
+Open a funding request issue using the most appropriate funding request issue
+template.
+
+After filing an issue please expect to wait 1 week for approval and potentially
+another week to a month for funding from the CNCF depending on the request size.
+
+## SIG or Committee Approvals
+
+Please provide links to show the appropriate approval process was followed by
+including links to the relevant approvals (meeting minutes, e-mail thread, etc.)
+
+## FAQ
+
+**What happens behind the scenes?**
+
+If the request is approved it will then go through the CNCF Service Desk
+however only a small number of people from the project can directly make
+requests.
+
+**Who Pays for this stuff?**
+
+The Kubernetes project relies on a number of paid services for day-to-day
+operations; this includes Zoom for video calls, Google cloud for test
+infrastructure, and Netlify for website deployments. Also, the project covers
+one-off costs like space for developer summits to bring developers together or
+swag incentivize and celebrate project successes. And these bills are paid for
+by CNCF members via their membership fees and at times in-kind donations.
+
+### Code of conduct
+
+Participation in the Kubernetes community is governed by the
+[Kubernetes Code of Conduct](code-of-conduct.md).
+
+[owners]: https://git.k8s.io/community/contributors/guide/owners.md
+[Creative Commons 4.0]: https://git.k8s.io/website/LICENSE
```

Comment:
We probably don't need this here. Remnants from the funding README.

Reply:
true, should have just removed it when copying over. Will fix quick

Reply:
FWIW: I generalize encourage a straight copy over then follow-ups in secondary commits or PRs because then it's obvious what changed but not a big deal here